eZ/Bundle/EzPublishCoreBundle/ApiLoader/RepositoryFactory.php

```diff
@@ -46,14 +46,21 @@ class RepositoryFactory extends ContainerAware
      */
     protected $roleLimitations = array();
 
+    /**
+     * @var array
+     */
+    private $limitationMap;
+
     public function __construct(
         ConfigResolverInterface $configResolver,
         FieldTypeCollectionFactory $fieldTypeCollectionFactory,
-        $repositoryClass
+        $repositoryClass,
+        array $limitationMap
     ) {
         $this->configResolver = $configResolver;
         $this->fieldTypeCollectionFactory = $fieldTypeCollectionFactory;
         $this->repositoryClass = $repositoryClass;
+        $this->limitationMap = $limitationMap;
```

Comment:
assuming this is what you refer to as `policyMap`, feel free to rename variable here, in Repository and Role Service if you want to keep it consistent.